README.md

```diff
@@ -88,12 +88,14 @@ While Golang, Docker and OVS can run on many operating systems, we are currently
 * Clustering over NAT adapter will not work. Hence, the Virtualbox VMs must have either **Host-Only Adapter (or) Internal Network (or) Bridged adapter** installed for clustering to work.
 * The VMs/Hosts must have **unique hostname**. Make sure that /etc/hosts in the VMs have the unique hostname updated.
 
-    curl -sSL http://get.socketplane.io/ | sudo sh
+    curl -sSL http://get.socketplane.io/ | sudo BOOTSTRAP=true sh
```

Comment:
Maybe worth adding it in this format

    # First Node
    curl -sSL http://get.socketplane.io/ | sudo BOOTSTRAP=true sh
    # Subsequent Nodes
    curl -sSL http://get.socketplane.io/ | sudo sh